### DISCUSSION TITLE
GF 660TI not supported?

### DISCUSSION THREAD

**Discussion by derWalter** [Rem0o/FanControl.Releases#1346] [Q&A]
where can I find out what GPUs are supported?

couldn't find a list

I am running win11 pro on a Asus Sabbertooth z77
with a GF 660TI and I can not controll the fanspeed of the GPU (greyed out,
but not hidden)
no other software installed to controll fanspeed

any idea why that is?

**Comment by derWalter** [comment]
no one has an idea, where to find a compatibility list, or what could be the culprit in my case? :)

**Reply by Rem0o** [reply]
Detected but grayed out means FanControl sees it but can't take over as the card doesn't respond to the input.   There is no compatibility list, as the backend is ever-evolving. 

![image](https://user-images.githubusercontent.com/14118956/206002574-a94ebc37-cf16-414e-a3c3-f68404f43edb.png)

You can try the alternative backend, by disabling the nvapi one and keeping the gpu section on the left, maybe you'll have better results there.

**Reply by derWalter** [reply]
Thanks for the answer, I forgot to mention that I ve already tried that.
The only thing that happened was that the name of the GPU disappeared,
but GPU is still greyed out.

Out of a sudden, I can controll it, but I cant get it to go below 1200rpm,
even though I ve ticked the Nvidia 0% hardware curve overide box ticked.

🤷

![image](https://user-images.githubusercontent.com/158642/206312208-86cfa785-547c-48a3-a431-e5aa1338ea8d.png)

played around with the values a bit, never goes under 1200rpm

**Reply by Rem0o** [reply]
Nvidia hard caps a lots of cards at 30% minimum, that's normal.

**Reply by derWalter** [reply]
😢
